src/components/taskManagement/tasks/Tasks.js

```diff
@@ -0,0 +1,128 @@
+import styles from "./tasks.module.css";
+import { useEffect, useState } from "react";
+import { v4 as uuid } from "uuid";
+import { StartTimerButton, EditTask, AddTag, FilterByTag } from "../../index";
+import { useTask } from "../../../contexts/index";
+import { GrEdit } from "react-icons/gr";
+import { AiFillDelete } from "react-icons/ai";
+import { MdCancel } from "react-icons/md";
+
+function Tasks(prop) {
+  const [input, setInput] = useState("");
+  const { state, dispatch } = useTask();
+  const { tasks } = state;
+
+  useEffect(() => {
+    localStorage.setItem("savedTasks", JSON.stringify(tasks));
```

Comment:
read/write operation on localstorage might fail, better to wrap this code in try/catch block

Reply:
will do